frontend/integration-tests/tests/kubevirt/non-admin.scenario.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-undef */
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { nonAdminSecret, nonAdminProvider } from './utils/mocks';
+import { click, createResources } from './utils/utils';
+import { KUBEADMIN_IDP, KUBEADMIN_USERNAME, BRIDGE_KUBEADMIN_PASSWORD, BRIDGE_HTPASSWD_IDP, BRIDGE_HTPASSWD_USERNAME, BRIDGE_HTPASSWD_PASSWORD } from './utils/consts';
+import * as loginView from '../../views/login.view';
+import * as crudView from '../../views/crud.view';
+import * as yamlView from '../../views/yaml.view';
+import { createWithWizardLink, createWithYAMLLink, closeWizard } from '../../views/kubevirt/wizard.view';
+import { logout } from '../../views/kubevirt/login.view';
+import { execSync } from 'child_process';
+
+describe('Test nonadmin user behaviour', () => {
+  const nonAdminNS = `${testName}-nonadmin`;
+  const userRole = ['view', 'edit'];
+
+  const verifyPermissions = async function( perm: string, ns: string) {
```

Comment:
What is the benefit of this to be an inner function?

Reply:
I see some same usage about the inner function in `modal-annotations.scenario.ts`, so just try to use the similar thing here. Is it so bad?

Reply:
It is not that bad (:, just it would be easier to potentially reuse this function in the future.